inc/targetbase.class.php

```diff
@@ -388,6 +390,55 @@ protected function setTargetCategory($data, $formanswer) {
          case self::CATEGORY_RULE_SPECIFIC:
             $category = $this->fields['category_question'];
             break;
+         case self::CATEGORY_RULE_LAST_ANSWER:
+            $category = null;
```

Comment:
a suggestion : move this line before the switch() {...} and drop the default case.

Reply:
Done.